src/KeycloakGuardServiceProvider.php

```diff
@@ -8,9 +8,11 @@ class KeycloakGuardServiceProvider extends ServiceProvider
 {
   public function boot()
   {
-    $this->publishes([
-      __DIR__ . '/../config/keycloak.php' => config_path('keycloak.php'),
```

Comment:
H@robsontenorio @zizou86 ,
we're currently working on a Lumen microservice as well and want to integrate it into Keycloak. As Lumen doesn't provide the helpers out of the box, we can just replace the `conflg_path` helper method with the content `app()->configPath()`.
It would be great if one of you can incorporate that line. Otherwise I will create a PR by tomorrow.

Regards

```suggestion
          __DIR__ . '/../config/keycloak.php' => app()->configPath('keycloak.php'),
```

Reply:
Hi @cunio-martin would be great if you submit a PR. Thanks.